pkg/scheduler/core/spreadconstraint/group_clusters.go

```diff
@@ -128,6 +128,46 @@ func groupClustersIgnoringTopology(
 	return groupClustersInfo
 }
 
+const weightUnit int64 = 1000
```

Comment:
to explain this `weightUnit`, pls add some comments here.

Reply:
ok, i'll do it.

Reply:
This const is used in the `calcGroupScore` method, why not put it in as a variable?

Reply:
Since the `weightUnit` variable represents a unit, I have defined it as a constant. Additionally, because `weightUnit` has a significant impact on the algorithm, I defined it outside the function to emphasize its importance. This also makes it easier to adjust in the future. @RainbowMango

Reply:
OK, then. no big deal.